repo/packages/A/arangodb3/5/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.8",
+  "name": "arangodb3",
+  "version": "3.1.x",
+  "scm": "https://github.com/arangodb/arangodb3-dcos.git",
+  "maintainer": "info@arangodb.com",
+  "description": "A distributed free and open-source database with a flexible data model for documents, graphs, and key-values. Build high performance applications using a convenient SQL-like query language or JavaScript extensions.",
```

Comment:
please add a link to the example at the end of `description`: Ex:
`Installation documentation can be found at https://github.com/dcos/examples/tree/master/1.8/arangodb`